File: Ejercicio 3/Ejercicio 3 (Indice determinado).cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        List<int> numeros = new List<int> { 10, 20, 30, 40, 50 };
        int valorAInsertar = 75;
        int indice = 2;

        if (indice >= 0 && indice <= numeros.Count)
        {
            numeros.Insert(indice, valorAInsertar);
        }
        else
        {
            Console.WriteLine("Índice fuera de rango.");
        }

        // Mostrar el resultado
        Console.WriteLine("Lista actualizada: " + string.Join(", ", numeros));
    }
}

File: Ejercicio 3/Ejercicio 3 (Indice determinado).py
numeros = [10, 20, 30, 40, 50]
valorAInsertar = 75
indice = 2

if 0 <= indice <= len(numeros):
    numeros.insert(indice, valorAInsertar)
else:
    print("Índice fuera de rango.")

# Mostrar el resultado (forma más simple)
print("Lista actualizada:", numeros)